test/functional/apps/dashboard/_dashboard_time.js

```diff
@@ -0,0 +1,70 @@
+import expect from 'expect.js';
+
+import { bdd } from '../../../support';
+import PageObjects from '../../../support/page_objects';
+
+const dashboardName = 'Dashboard Test Time';
+
+bdd.describe('dashboard time', function dashboardSaveWithTime() {
+  bdd.before(async function () {
+    return PageObjects.dashboard.initTests();
```

Comment:
should be `await` instead of `return` in async method?  Maybe doesn't matter if it's the only call?